Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2273,15 +2314,19 @@ - (void)_completeTextManipulation:(_WKTextManipulationItem *)item completion:(vo
         return;
     }
 
-    auto itemID = coreTextManipulationItemIdentifierFromString(item.identifier);
+    auto identifiers = coreTextManipulationItemIdentifierFromString(item.identifier);
+    if (!identifiers) {
+        completionHandler(false);
+        return;
+    }
 
     Vector<WebCore::TextManipulationToken> tokens;
     for (_WKTextManipulationToken *wkToken in item.tokens)
         tokens.append(WebCore::TextManipulationToken { coreTextManipulationTokenIdentifierFromString(wkToken.identifier), wkToken.content, std::nullopt });
 
     Vector<WebCore::TextManipulationItem> coreItems;
     coreItems.reserveInitialCapacity(1);
-    coreItems.uncheckedAppend(WebCore::TextManipulationItem { itemID, WTFMove(tokens) });
+    coreItems.uncheckedAppend(WebCore::TextManipulationItem { identifiers->frameID, false, false, identifiers->itemID, WTFMove(tokens) });
```

Comment:
I understand we don't need `isSubFrame` info for completeTextManipulation, but it feels a bit weird to convert it to false when we know the real value